contracts/SimpleCertifier.sol

```diff
@@ -52,7 +52,7 @@ contract SimpleCertifier is Owned, Certifier {
 		return certs[_who].active;
 	}
 
-	function get(address _who, string _field) view public returns (bytes32) {
+	function getData(address _who, string _field) view public returns (bytes32) {
```

Comment:
And this is a potentially-breaking change in ABI — would require not only re-deployment of contracts and changing the address in the clients, but also changing instrumentation in the clients themselves.

I support the new name from the aesthetical point of view, but let's carefully consider this decision.

Reply:
This was actually from our audit report issue `#6`. I think it's a typo in the original contract -- otherwise the interface in `Certifier` would simply not be implemented.